airbyte-integrations/connectors/source-snowflake/unit_tests/integration/request_builder.py

```diff
@@ -72,6 +74,15 @@ def with_where_statement(self, where_statement: str):
         self._where_statement = where_statement
         return self
 
+    def with_state(self, state):
```

Comment:
corrected in this PR https://github.com/artefactory/airbyte/pull/45 because the code is refactored. No more concepts of state and cursor in request builder.